sale_medical_prescription/wizards/medical_sale_wizard.py

```diff
@@ -110,12 +114,14 @@ def action_create_sale_wizards(self, ):
             client_order_ref = ', '.join(
                 p.name for p in prescription_order_ids
             )
+
+            partner_id = patient_id.partner_id
             order_inserts.append((0, 0, {
-                'partner_id': patient_id.partner_id.id,
+                'partner_id': partner_id.id,
                 'patient_id': patient_id.id,
                 'pricelist_id': pricelist_id,
-                'partner_invoice_id': patient_id.id,
```

Comment:
@lasley these were changed from `patient_id.id` to `partner_id.id`. During testing I had initially tested against patients, however the tests showed the results were pointing to related partner of patient.